gdb/__init__.py

```diff
@@ -936,21 +935,29 @@ def lookup_type(name, block=None):
     raise error('Type "%s" not found in %d matches.' % (name, count))
 
 
-def current_objfile():
+class Objfile:
+    pass
+
+
+def current_objfile() -> Optional[Objfile]:
```

Comment:
Maybe just `-> None`?

Reply:
According to https://sourceware.org/gdb/onlinedocs/gdb/Objfiles-In-Python.html

> When auto-loading a Python script (see [Python Auto-loading](https://sourceware.org/gdb/onlinedocs/gdb/Python-Auto_002dloading.html#Python-Auto_002dloading)), GDB sets the “current objfile” to the corresponding objfile. This function returns the current objfile. If there is no current objfile, this function returns None.

and also:

> Each objfile is represented by an instance of the gdb.Objfile class.

The `class Objfile` in this source file is GALA's implementation of `gdb.Objfile`. And `current_objfile()` is supposed to return an object of that class, or `None`. Right now we always return `None` because this feature is not implemented, but we could add it to the autoload module.

Reply:
Done.